src/commands/org/create/sandbox.ts

```diff
@@ -175,6 +175,22 @@ export default class CreateSandbox extends SandboxCommandBase<SandboxCommandResp
       delete sandboxReq.ActivationUserGroupName;
     }
 
+    // Get source sandbox features if cloning
+    if (srcSandboxName) {
+      const sourceFeatures = await requestFunctions.getFeature(
+        this.flags['target-org'].getConnection(),
+        srcSandboxName
+      );
+      if (sourceFeatures) {
+        (sandboxReq as SandboxRequest & { Features?: string }).Features = sourceFeatures;
+      }
+    } else if (srcId) {
+      const sourceFeatures = await requestFunctions.getFeature(this.flags['target-org'].getConnection(), srcId);
+      if (sourceFeatures) {
+        (sandboxReq as SandboxRequest & { Features?: string }).Features = sourceFeatures;
+      }
+    }
+
```

Comment:
1. We should update the types for `SandboxRequest`, `SandboxInfo`, and `SandboxProcess` in sfdx-core repo so we don't have to do the intersection type here and other places.
2. There's duplicate code in the conditionals that could be simplified.  Maybe something like, `const nameOrId = srcSandboxName ?? srcId;` then if `nameOrId` is defined call getFeature()